fw/sock.c

```diff
@@ -18,6 +18,12 @@
  * this program; if not, write to the Free Software Foundation, Inc., 59
  * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
+#undef DEBUG
+#if DBG_SS > 0
+#define DEBUG DBG_SS
+#endif
+#include "lib/log.h"
+
```

Comment:
That's dangerous and seems lead to https://github.com/tempesta-tech/tempesta/issues/2163 : the Linux kernel headers also use `DEBUG` : 
```
linux-5.10.35-tfw$ grep -r '\<DEBUG\>' include/linux/*
include/linux/acpi.h:#if defined(DEBUG)
include/linux/ceph/ceph_debug.h:# if defined(DEBUG) || defined(CONFIG_DYNAMIC_DEBUG)
include/linux/dev_printk.h:#elif defined(DEBUG)
include/linux/dev_printk.h:#elif defined(DEBUG)
include/linux/dynamic_debug.h:#if defined DEBUG
include/linux/dynamic_debug.h:#ifdef DEBUG
include/linux/dynamic_debug.h:#ifdef DEBUG
include/linux/mISDNhw.h: * HW DEBUG 0xHHHHGGGG
include/linux/mtd/rawnand.h:#if IS_ENABLED(CONFIG_DYNAMIC_DEBUG) || defined(DEBUG)
include/linux/netdevice.h:#elif defined(DEBUG)
include/linux/netdevice.h:#elif defined(DEBUG)
include/linux/net.h:#elif defined(DEBUG)
include/linux/platform_data/dma-iop32x.h:#ifdef DEBUG
include/linux/printk.h: * This macro expands to a printk with KERN_DEBUG loglevel if DEBUG is
include/linux/printk.h:#ifdef DEBUG
include/linux/printk.h: * set. Otherwise, if DEBUG is defined, it's equivalent to a printk with
include/linux/printk.h: * KERN_DEBUG loglevel. If DEBUG is not defined it does nothing.
include/linux/printk.h:#elif defined(DEBUG)
include/linux/printk.h:#if defined(DEBUG)
include/linux/printk.h:#if defined(DEBUG)
include/linux/printk.h:#if defined(DEBUG)
include/linux/printk.h:#elif defined(DEBUG)
include/linux/printk.h:#elif defined(DEBUG)
include/linux/skbuff.h:#ifdef DEBUG
include/linux/slab.h: * The ones marked DEBUG are only valid if CONFIG_DEBUG_SLAB is set.
include/linux/slab.h:/* DEBUG: Perform (expensive) checks on alloc/free */
include/linux/slab.h:/* DEBUG: Red zone objs in a cache */
include/linux/slab.h:/* DEBUG: Poison objects */
include/linux/slab.h:/* DEBUG: Store the last owner for bug hunting */
include/linux/ssb/ssb.h:#endif /* DEBUG */
include/linux/timer.h: * LOCKDEP and DEBUG timer interfaces.
include/linux/vbox_utils.h:#if defined(DEBUG) && !defined(CONFIG_DYNAMIC_DEBUG)
include/linux/wimax/debug.h: * DEBUG LEVELS
```

Reply:
Yes fixed